src/tests/ftest/pool/create_capacity_test.yaml

```diff
@@ -24,4 +24,7 @@ server_config:
 pool:
   name: daos_server
   control_method: dmg
+  scm_size: 60%
+  nvme_size: 60%
```

Comment:
So this will create 200 pools out of 60% of the capacity?

Reply:
Yes, the combination of `quantity: 200` with `scm_size: 60%` and `nvme_size: 60%` will yield a single pool with a `dmg pool create` command with `--scm_size=~(<SCM_per_engine> * 0.6) / 200` and `nvme_size=~(<NVMe_per_engine> * 0.6) / 200`.  Then if the code calls `self.add_pool_qty(200)` then 200x pools will be created with these sizes:
```
2021-06-25 23:04:08,086 server_utils     L0820 INFO | ----------------------------------------------------------------------------------------------------
2021-06-25 23:04:08,086 server_utils     L0824 INFO | Autosizing TestPool parameters ending with a "%" for 200 pools:
2021-06-25 23:04:08,086 server_utils     L0827 INFO |   - size      : None (False)
2021-06-25 23:04:08,086 server_utils     L0827 INFO |   - scm_size  : 60% (True)
2021-06-25 23:04:08,086 server_utils     L0827 INFO |   - nvme_size : 60% (True)
2021-06-25 23:04:08,086 server_utils_bas L0502 INFO | Detected device capacities:
2021-06-25 23:04:08,086 server_utils_bas L0509 INFO |   NVME for 0000:86:00.0 : ['750156374016 (698.64GiB) (750.16GB)']
2021-06-25 23:04:08,086 server_utils_bas L0509 INFO |   NVME for 0000:87:00.0 : ['750156374016 (698.64GiB) (750.16GB)']
2021-06-25 23:04:08,086 server_utils_bas L0509 INFO |   SCM  for pmem0 : ['3183575302144 (2.9TiB) (3.18TB)']
2021-06-25 23:04:08,086 server_utils_bas L0509 INFO |   SCM  for pmem1 : ['3183575302144 (2.9TiB) (3.18TB)']
2021-06-25 23:04:08,086 server_utils_bas L0536 INFO | Detected engine capacities:
2021-06-25 23:04:08,087 server_utils_bas L0540 INFO |   NVME : ['1500312748032 (1.36TiB) (1.5TB)']
2021-06-25 23:04:08,087 server_utils_bas L0540 INFO |   SCM  : ['3183575302144 (2.9TiB) (3.18TB)']
2021-06-25 23:04:08,087 server_utils     L0755 INFO | Largest storage size available per engine:
2021-06-25 23:04:08,087 server_utils     L0760 INFO |   NVME:  1500312748032 (1.36TiB) (1.5TB)
2021-06-25 23:04:08,087 server_utils     L0760 INFO |   SCM :  3183575302144 (2.9TiB) (3.18TB)
2021-06-25 23:04:08,087 server_utils     L0851 INFO | Largest storage size available for 7 engines with a 6.00% scm_ratio:
2021-06-25 23:04:08,087 server_utils     L0854 INFO |   - NVME     : 10502189236224 (9.55TiB) (10.5TB)
2021-06-25 23:04:08,087 server_utils     L0857 INFO |   - SCM      : 630131354173.44 (586.86GiB) (630.13GB)
2021-06-25 23:04:08,087 server_utils     L0861 INFO |   - COMBINED : 11132320590397.44 (10.12TiB) (11.13TB)
2021-06-25 23:04:08,087 server_utils     L0869 INFO | Adjusted pool sizes for 200 pools:
2021-06-25 23:04:08,088 server_utils     L0882 INFO |   - scm_size  : SCM  storage adjusted by 60.00%: 9550725906.432 (8.89GiB) (9.55GB)
2021-06-25 23:04:08,088 server_utils     L0882 INFO |   - nvme_size : NVMe storage adjusted by 60.00%: 4500938244.096 (4.19GiB) (4.5GB)
2021-06-25 23:04:08,088 server_utils     L0889 INFO | Increment sizes per target:
2021-06-25 23:04:08,088 server_utils     L0892 INFO |   - scm_size  : 16777216 (16.0MiB) (16.78MB)
2021-06-25 23:04:08,088 server_utils     L0892 INFO |   - nvme_size : 1073741824 (1024.0MiB) (1.07GB)
2021-06-25 23:04:08,088 server_utils     L0895 INFO | Pool sizes adjusted to fit by increment sizes:
2021-06-25 23:04:08,088 server_utils     L0903 INFO |   - scm_size  : 569 * 16777216 = 9546235904 (8.89GiB) (9.55GB)
2021-06-25 23:04:08,089 server_utils     L0903 INFO |   - nvme_size : 4 * 1073741824 = 4294967296 (4.0GiB) (4.29GB)
2021-06-25 23:04:08,089 server_utils     L0931 INFO | ----------------------------------------------------------------------------------------------------
2021-06-25 23:04:08,089 command_utils_ba L0116 DEBUG| Updated param scm_ratio => None
2021-06-25 23:04:08,089 command_utils_ba L0116 DEBUG| Updated param scm_size => 8.89GiB
2021-06-25 23:04:08,089 command_utils_ba L0116 DEBUG| Updated param nvme_size => 4.0GiB
2021-06-25 23:04:08,110 test_utils_pool  L0144 INFO | Creating a pool
2021-06-25 23:04:08,111 general_utils    L0260 INFO | Command environment vars:
  None
2021-06-25 23:04:08,111 process          L0626 INFO | Running '/usr/bin/dmg -o /etc/daos/daos_control.yml -d -j pool create --group=jenkins --nsvc=1 --nvme-size=4.0GiB --scm-size=8.89GiB --user=jenkins'
```